cle/Chart.yaml

```diff
@@ -1,8 +1,8 @@
 apiVersion: v1
-appVersion: "0.2.7"
+appVersion: "0.2.0"
```

Comment:
is this intentional

Reply:
Yes @ksatchit , This is updated as discussed with @rajdas98. It should have been 0.2.0 instead of 0.2.7. We have fixed it now.